source/modulo_components/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>modulo_components</name>
-  <version>2.1.7</version>
+  <version>2.1.8</version>
   <description>TODO</description>
```

Comment:
As we are here, should we remove this TODO and actually provide a small description here?